$OpenBSD$

OpenBSD path and there is no point in logging to stderr if we use daemon() to daemonise
(stderr will get closed).

Fix a typo.

--- LCDd.conf.orig	Wed Nov  9 19:50:01 2011
+++ LCDd.conf	Tue Nov 15 17:16:25 2011
@@ -34,7 +34,7 @@
 #            the driver modules and will thus not be able to
 #            function properly.
 # NOTE: Always place a slash as last character !
-DriverPath=server/drivers/
+DriverPath=${LOCALBASE}/lib/lcdproc/
 
 # Tells the server to load the given drivers. Multiple lines can be given.
 # The name of the driver is case sensitive and determines the section
@@ -62,11 +62,11 @@ Port=13666
 #ReportLevel=3
 
 # Should we report to syslog instead of stderr ? [default: no; legal: yes, no]
-#ReportToSyslog=yes
+ReportToSyslog=yes
 
 # User to run as.  LCDd will drop its root privileges, if any,
 # and run as this user instead.
-User=nobody
+User=_lcdproc
 
 # The server will stay in the foreground if set to true.
 #Foreground=no
@@ -79,7 +79,7 @@ User=nobody
 #GoodBye="Thanks for using"
 #GoodBye="   LCDproc!"
 
-# Sets the default time in seconds to displays a screen.
+# Sets the default time in seconds to display a screen.
 WaitTime=5
 
 # If set to no, LCDd will start with screen rotation disabled. This has the
